guides/managing-current-user.md

```diff
@@ -162,3 +151,28 @@ export default class ApplicationRoute extends Route.extend(ApplicationRouteMixin
   }
 });
 ```
+
+In the latter case Ember Simple Auth will call the session service's
+`handleAuthentication` method. The `currentUser` service's `load` method must
+be called in that cases as well. We can do that by overriding the session
+sevice's method in a custom extension of Ember Simple Auth's standard session
+service:
+
+```js
+import { inject as service } from '@ember/service';
+import SessionService from 'ember-simple-auth/services/session';
+
+export default SessionService.extend({
+  @service currentUser;
```

Comment:
This is using class decorator syntax but not a class. Would submit a pr but on my phone.

Reply:
thanks – see #2210